prow/github/client.go

```diff
@@ -1511,15 +1511,22 @@ func (c *Client) ListTeams(org string) ([]Team, error) {
 }
 
 // ListTeamMembers gets a list of team members for the given team id
-func (c *Client) ListTeamMembers(id int) ([]TeamMember, error) {
-	c.log("ListTeamMembers", id)
+//
+// Role options are "all", "maintainer" and "member"
```

Comment:
Can we get a type alias for this and some constants?

Reply:
Constants added